src/jrd/jrd.h

```diff
@@ -485,7 +485,6 @@ const ULONG TDBB_use_db_page_space		= 128;		// use database (not temporary) page
 const ULONG TDBB_detaching				= 256;		// detach is in progress
 const ULONG TDBB_wait_cancel_disable	= 512;		// don't cancel current waiting operation
 const ULONG TDBB_cache_unwound			= 1024;		// page cache was unwound
-const ULONG TDBB_trusted_ddl			= 2048;		// skip DDL permission checks. Set after DDL permission check and clear after DDL execution
```

Comment:
Please move the other constants' values.

Reply:
OK